src/store/vmApi.ts

```diff
@@ -51,80 +50,72 @@ export const getActualPlayerState = async (dispatch: React.Dispatch<action>, nod
  * Fetch the state from /vm-state and return the state ordered by DESC amount of cookies
  */
 const getRawLeaderBoard = async (nodeUri: string): Promise<any> => {
-    const stateRequest = await fetch(nodeUri + "/vm-state",
+    const stateRequest = await fetch(nodeUri + "/api/v1/state/unix/",
         {
-            method: "POST",
-            body: JSON.stringify(null)
+            method: "GET"
         });
     const stateResponse = JSON.parse(await stateRequest.text(), parseReviver);
-    const value = stateResponse.state.sort(((n1: any, n2: any) => {
-        const baker1: cookieBaker = JSON.parse(n1[1], parseReviver);
-        const baker2: cookieBaker = JSON.parse(n2[1], parseReviver);
-        // we want DESC ordering
-        return Number(baker2.eatenCookies - baker1.eatenCookies)
-    }));
-    return value;
+    const sorted =
```

Comment:
@Pilou97 it looks like this is not sorting :/
Do you have any idea on how to solve this bug?
cc @d4hines

Reply:
The sort function does not return a new array it mutated the given one